tvb_build/docker/Dockerfile-build

```diff
@@ -40,7 +40,7 @@ RUN /bin/bash -c "source activate tvb-run"; \
     /opt/conda/envs/tvb-run/bin/jupyter notebook --generate-config; \
     echo "c.NotebookApp.password='sha1:12bff019c253:9daecd92c2e9bdb10b3b8a06767a74a0fe078d7c'">>$JUPYTER_CONFIG/jupyter_notebook_config.py
 
-RUN wget https://zenodo.org/record/10128131/files/tvb_data.zip?download=1 -O tvb_data.zip; \
+RUN wget https://zenodo.org/record/14992335/files/tvb_data.zip?download=1 -O tvb_data.zip; \
     mkdir tvb_data; unzip tvb_data.zip -d tvb_data; rm tvb_data.zip; \
     cd tvb_data; \
     /opt/conda/envs/tvb-run/bin/python setup.py develop;\
```

Comment:
The install of the new `tvb-data` should be adjusted from usage of the old `setup.py` to probably `pip install -e .`

Reply:
yes, was waiting for all tests to finish to do the update

Reply:
line 46 should be changed then here